plugins/base/src/main/resources/dokka/scripts/platformContentHandler.js

```diff
@@ -7,24 +7,47 @@ window.addEventListener('load', () => {
     document.querySelectorAll("div[data-platform-hinted]")
         .forEach(elem => elem.addEventListener('click', (event) => togglePlatformDependent(event,elem)))
     document.querySelectorAll("div[tabs-section]")
-        .forEach(elem => elem.addEventListener('click', (event) => toggleSections(event)))
+        .forEach(elem => elem.addEventListener('click', (event) => toggleSectionsEventHandler(event)))
     document.getElementById('filter-section').addEventListener('click', (event) => filterButtonHandler(event))
     initializeFiltering()
     initTabs()
+    handleAnchor()
 })
 
+function handleAnchor() {
+    let searchForTab = function(element) {
+        if(element && element.hasAttribute) {
```

Comment:
what `element.hasAttribute` does in this case?
imho as it is an existing property it evaluates to true